app/controllers/api/v1/monsters_controller.rb

```diff
@@ -1,4 +1,7 @@
 class Api::V1::MonstersController < ApplicationController
+  before_action :authenticate_user, only: [:create, :delete]
+  before_action :authorize_user, only: [:delete]
+  before_action :authenticate_user!, except: [:index, :show]
```

Comment:
haha we shouldnt have TWO authenticate_user methods being called here I think? or am I missing something? The one with the `!` I believe is the default one from rails yeah?